packages/frontend/src/types/index.ts

```diff
@@ -6,22 +6,25 @@ export type ItemId = `0x${string}`;
 export type TagId = `0x${string}`;
 export type OrderId = `0x${string}`;
 
-export type IProduct = {
-  id: ItemId;
+export interface Item {
```

Comment:
These should be removed in favour of the definitions in the state manager

https://github.com/masslbs/Tennessine/blob/a57bdd9f91daa81fd3ded1ceb60dc3e1e8c119fd/packages/stateManager/index.ts#L29-L86